### PR TITLE
docs(changelog): 5.3 release

### PR DESCRIPTION
**Pull request by P0lip**
~Release is blocked by https://github.com/stoplightio/spectral/pull/1051~

**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,14 +7,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.3.0] - 2019-04-01
+
+### Added
+- Built-in functions can now be accessed in custom functions [#925](https://github.com/stoplightio/spectral/pull/925)
```
Comment (nulltoken) [inline CHANGELOG.md:13]:
```suggestion
- Built-in functions can now be accessed from custom functions [#925](https://github.com/stoplightio/spectral/pull/925)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,14 +7,30 @@ All notable changes to this project will be documented in this file.
...
+- CLI: `--show-unmatched-globs` flag  [#747](https://github.com/stoplightio/spectral/issues/747)
+
+### Changed
+- `typed-enum` function is a part of the OpenAPI ruleset [#924](https://github.com/stoplightio/spectral/issues/924)
```
Comment (nulltoken) [inline CHANGELOG.md:17]:
```suggestion
- `typed-enum` function is no longer a built-in function. It's been migrated into a custom function of the OpenAPI ruleset [#924](https://github.com/stoplightio/spectral/issues/924)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,14 +7,30 @@ All notable changes to this project will be documented in this file.
...
+- `openapi-tags` rule fit its description [#1038](https://github.com/stoplightio/spectral/pull/1038)
```
Comment (nulltoken) [inline CHANGELOG.md:18]:
```suggestion
- `openapi-tags` rule has been fixed to make it fit its description [#1038](https://github.com/stoplightio/spectral/pull/1038)
```

**Comment by P0lip** [comment]
@philsturgeon @nulltoken feedback applied. LMK if we are good.

**Pull request merged**